<a href="https://colab.research.google.com/github/H2D2R2/Dz4/blob/main/%D0%94%D0%BE%D0%BC%D0%B0%D1%88%D0%BD%D0%B5%D0%B5_%D0%B7%D0%B0%D0%B4%D0%B0%D0%BD%D0%B8%D0%B5_4.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Задание 1 

Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

In [3]:
A = 200
B = 800
M = (A+B)/2
M

500.0

In [5]:
D = (B-A)**2/12
D

30000.0

Ответ: среднее значение - 500, дисперсия - 500

# Задание 2

О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.

In [7]:
D = 0.2
a = 0.5
b = 0.5 + (0.2*12)**1/2
b

1.7000000000000002

In [8]:
m = (a+b)/2
m

1.1

Ответ: Правая граница величины - 1,7; Среднее значение 1.1

# Задание 3

Непрерывная случайная величина X распределена нормально и задана плотностью распределения f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-((x+2)**2) / 32))
Найдите:
а). M(X)
б). D(X)
в). std(X) (среднее квадратичное отклонение)

In [9]:
M = -2
s = 4
D = s **2 
D

16

Ответ: Математическое ожидание -2; Cреднее отклонение 4; Дисперсия 16;

# Задание 4

Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
а). больше 182 см
б). больше 190 см
в). от 166 см до 190 см
г). от 166 см до 182 см
д). от 158 см до 190 см
е). не выше 150 см или не ниже 190 см
ё). не выше 150 см или не ниже 198 см
ж). ниже 166 см.

In [17]:
s1 = 34.1
s2 = 13.6
s3 = 2.15
s4 = 0.15

A = s2+s3+s4
A

15.9

In [18]:
B = s3+s4
B

2.3

In [19]:
V = s1+s1+s2
V

81.8

In [20]:
G = s1*2
G

68.2

In [21]:
D = s1*2 + s2*2
D

95.4

In [22]:
E = s4*2 + s3
E

2.4499999999999997

In [23]:
EO = s4*2
EO

0.3

In [24]:
J = s2+s3+s4
J

15.9

Ответ: А)15.9 Б)2.3 B)81.8 Г)68.2 Д)95.4 Е)2.4 Ё)0.3 Ж)15.9

# Задание 5

На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [25]:
D = 25; std = 5
M = 178
X = 190

Nsigm = (X-M)/std
Nsigm

2.4

Ответ: 2.4 сигмы